source/test/TestApp/TestApp.cpp.ejs

```diff
@@ -53,13 +53,16 @@ PFTestTraceLevel TestApp::traceLevel = PFTestTraceLevel::Important;
         LogPut(message);
     }
 #endif
+    
+    std::unique_ptr<PFTestData> g_data;
```

Comment:
>std::unique_ptr<PFTestData> g_data; [](http://example.com/codeflow?start=4&length=35)

Any consideration for passing prerequisite inputs to individual tests rather than stuffing everything into a global struct?  This looks fine for now, but as it grows it can get a bit messy and hard to tell what global data is used where, which tests have prerequisite data, etc.

Don't necessarily need to change as part of this PR, just something to think about before we get to far along with this pattern.